portfolio/src/main/webapp/index.html

```diff
@@ -51,5 +52,72 @@ <h3 class="project-name">Flappy Bird</h3>
             <p class="tech-used">Technology Used: Python and the Pygame module.</p>
         </div>
       </div>
+      <div id="content">
+        <h2 class="header" id="gallery">Gallery</h2>
```

Comment:
Optional: there are a class and id named as gallery, feel free to change one of it to avoid confusion.